### ISSUE TITLE
Mocks for input errors

### ISSUE THREAD

**Issue by ssandbekkhaug** [civiform/civiform#8098]
[Mocks in progress](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=1696-2397&t=kN0JEf33OZIyFetw-0)

**Comment by sijiayam** [comment]
@ssandbekkhaug Do you have more details on what is needed here, what is missing

**Comment by ssandbekkhaug** [comment]
I guess the question is:

- For each question type in North Star, are we happy with the error UX?
- If not, what should the UX be?

If we're happy with the existing UX, we can close this issue.

**Comment by isabelcshaw** [comment]
<img width="429" alt="Screenshot 2024-10-15 at 9 08 54 AM" src="https://github.com/user-attachments/assets/8591535f-c5a2-4bc6-a441-0bcc491c3f53">

@ssandbekkhaug cc @sijiayam my understanding is that for if we trigger an error on any question (required but not filled out, incorrect input type, etc) we use this component above that shows a red box, red vertical line along the question and adds red error text below the helper text with "Error: [specific error]." If that is true, then I'm happy with the existing UX and we can close this issue. If not, let's talk about the smallest possible thing we can do here.

**Comment by sijiayam** [comment]
@isabelcshaw Agreed. I think it may be easier to test out the current error messages Greg implemented to see if there is anything that is broken UX? And then prioritize and decide if it is a pre-launch or post-launch issue. 

@shreyachatterjee00 @nb1701 Could this be a focus of one of the upcoming bug bashes?

I think we can close this issue.
